sdk/search/search-documents/src/index.ts

```diff
@@ -240,6 +240,16 @@ export {
   IndexerExecutionEnvironment,
   BlobIndexerImageAction,
   BlobIndexerParsingMode,
-  BlobIndexerPDFTextRotationAlgorithm
+  BlobIndexerPDFTextRotationAlgorithm,
+  Similarity,
+  CognitiveServicesAccount as CognitiveServicesAccountBase,
```

Comment:
This is a concerning change that is happening. In the previous versions, it was not required. But, since the object hierarchy is changed with the new generator, this code needs to be added.

Reply:
Yeah this feels like a lot extra exports... are all these `Base` types really necessary?